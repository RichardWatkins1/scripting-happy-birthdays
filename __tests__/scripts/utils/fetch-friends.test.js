const { fetchFriends } = require("../../../scripts/utils/fetch-friends")

describe("fetchFriends", () => {
  it("returns a list of friends", async () => {
    const results = await fetchFriends()
    expect(results).toMatchSnapshot()
  })
})