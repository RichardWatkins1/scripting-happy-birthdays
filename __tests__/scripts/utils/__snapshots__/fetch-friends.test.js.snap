// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchFriends returns a list of friends 1`] = `
Array [
  Object {
    "contact": "john.doe@foobar.com",
    "dob": "1982/10/08",
    "firstName": "John",
    "lastName": "Doe",
    "messageType": "email",
  },
  Object {
    "contact": "mary.ann@foobar.com",
    "dob": "1975/09/11",
    "firstName": "Mary",
    "lastName": "Ann",
    "messageType": "email",
  },
  Object {
    "contact": "bill.shatner@example.com",
    "dob": "1931/03/20",
    "firstName": "William",
    "lastName": "Shatner",
    "messageType": "email",
  },
]
`;
